pac4j-oauth/src/main/java/org/pac4j/oauth/profile/generic/GenericOAuth20Profile.java

```diff
@@ -0,0 +1,89 @@
+package org.pac4j.oauth.profile.generic;
+
+import org.pac4j.core.context.Pac4jConstants;
+import org.pac4j.core.profile.Gender;
+import org.pac4j.oauth.profile.OAuth20Profile;
+
+import java.util.Locale;
+
+/**
+ * <p>This is the user profile for the generic OAuth 2.0 client.</p>
+ * <p>It is returned by the {@link org.pac4j.oauth.client.GenericOAuth20Client}  </p>
+ *
+ * @author aherrick
+ * @since 1.9.2
+ */
+public class GenericOAuth20Profile extends OAuth20Profile {
+
+    private transient GenericAttributesDefinition attributes = new GenericAttributesDefinition();
```

Comment:
So as it turns out, it doesn't work as well as I thought it would. The Maven build tool won't allow me to set the GenericAttributesDefinition variables to non-final, and I know nothing about Maven, so I haven't figured out how to disable that for that file yet.

Using profile.getAttribute() still allows you to get all the values, but it's less user-friendly